### PR TITLE
Add IOTA

### PR DESCRIPTION
**Pull request by VladimirLevv**

### CODE REVIEW

**Comment by akhavr** [comment]
Please, resolve the conflicts.

Do we still need `templates/project_list.md` if we're using issues?

**Comment by VladimirLevv** [comment]
I guess, no, we don't need to leave any marks there

File: templates/project_list.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 5.  [Dash](/projects/Dash.md)
 6.  [NEM](/projects/NEM.md)
 7.  [Ethereum Classic](/projects/EthereumClassic.md)
-8.  [IOTA](/projects/IOTA.md)
+9.  [Monero](/projects/Monero.md)
```
Comment (akhavr) [inline templates/project_list.md]: Strange change.  Was Monero missing last time?

Perhaps, just create issue for Monero and delete this file completely?

Comment (VladimirLevv) [inline templates/project_list.md]: wer already have monero added, it should be deleted from this file

**Comment by akhavr** [comment]
Closes  #97

**Pull request merged**